tests/testthat/test_escaping.r

```diff
@@ -1,5 +1,9 @@
 context('LaTeX and HTML escaping')
 
+has_dt <- requireNamespace('data.table', quietly = TRUE)
+has_dplyr <- requireNamespace('dplyr', quietly = TRUE)
```

Comment:
Just a heads up: that will/should become tibble after the next version if dplyr... https://blog.rstudio.org/2016/03/24/tibble-1-0-0/